docs/reference/docs/data-replication.asciidoc

```diff
@@ -42,6 +45,14 @@ The primary shard follows this basic flow:
 . Once all replicas have successfully performed the operation and responded to the primary, the primary acknowledges the successful
    completion of the request to the client.
 
+Each in-sync replica copy performs the indexing operation locally so that it has a copy. This stage of indexing is the replication
+stage.
```

Comment:
```suggestion
Each in-sync replica copy performs the indexing operation locally so that it has a copy. This stage of indexing is the _replica
stage_.
```